iree/samples/simple_embedding/device_cuda.c

```diff
@@ -0,0 +1,47 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A example of setting up the the cuda driver.
```

Comment:
style: consistently use CUDA and HAL in commands

Reply:
fixed

Reply:
I'm still seeing `cuda` and both `hal` and `HAL` in here - did you forget to push the changes?